### PR TITLE
Add solution to Acronym

### PR DESCRIPTION
**Pull request by joshuaswickirl**

### CODE REVIEW

**Comment by joshuaswickirl** [comment]
@abcarp watchathink?

File: acronym/Acronym.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+public static class Acronym
+{
+    private static char[] delimiters = {' ', '-', '_'};
+
+    public static string Abbreviate(string phrase)
+    {
+        List<char> acronym = new List<char>();
+        string[] tokens = phrase.Split(delimiters);
+        foreach (string t in tokens)
```
Comment (abcarp) [inline acronym/Acronym.cs:13]: Passing StringSplitOptions.RemoveEmptyEntries as the second param in Split() would remove the need to check for empty strings later.

**Review by abcarp — APPROVED** [review]
I like the approach. Just the one suggestion.

File: acronym/Acronym.cs (hunk 1 above)
Comment (joshuaswickirl) [reply acronym/Acronym.cs:13]: Ooo nice. Thanks!

**Pull request merged**